Elf/__init__.py

```diff
@@ -517,10 +517,14 @@ def getNotes(self):
                 continue
 
             notebytes =  self.readAtOffset(sec.sh_offset, sec.sh_size)
-
             offset = 0
-            while offset < len(notebytes):
+            notebyteslen = len(notebytes)
+            while offset < notebyteslen:
                 note = vs_elf.ElfNote()
+                if notebyteslen - offset < len(note):
+                    print ("\nNOTES section length mismatch!\n\t%s\n\tSection Bytes: %s\n\tStranded bytes: %s\n" % (sec, repr(notebytes), repr(notebytes[offset:])))
```

Comment:
internal discussions.  i'm not going to step out and introduce this technique in this PR.  however, if it makes sense to do so for the overall project, i'll make the change here as well.